### ISSUE TITLE
Overfaced vertebral pedicle

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11699]
**Preferred term label:**
Overfaced vertebral pedicle

**Synonyms**
Overfaced vertebral pedicles

**Definition (free text, please give PubMed ID)**
An abnormal orientation of the vertebral pedicle. The vertebral arch is formed by pedicles and laminae. Two pedicles extend from the sides of the vertebral body to join the body to the arch. An overfaced vertebral pedicle is angled more medially than normal, which implies that the the lateral border of the vertebrae appears outside the lateral edge of the pedicles on the frontal view on spine radiography.
PMID: 37053103

**Parent term (use hpo.jax.org/app)**
Abnormal vertebral pedicle morphology 
HP:0030277

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Spondylometaphyseal dysplasia, Kozlowski type | 184252
PMID: 19232556 6/6 (phenopacket store)

Metatropic dysplasia | 156530
PMID: 19232556 0/2 (phenopacket store)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001530
- label: Overfaced vertebral pedicle
- parents: [Abnormal vertebral pedicle morphology (HP:0030277)](https://hpo.jax.org/app/browse/term/HP:0030277)
- pmid: PMID:37053103
- def: An abnormal orientation of the vertebral pedicle. The vertebral arch is formed by pedicles and laminae. Two pedicles extend from the sides of the vertebral body to join the body to the arch. An overfaced vertebral pedicle is angled more medially than normal, which implies that the the lateral border of the vertebrae appears outside the lateral edge of the pedicles on the frontal view on spine radiography.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11699
- synonyms: Overfaced vertebral pedicles(NONE)
